modules/tensor_mechanics/src/kernels/StressDivergenceTensors.C

```diff
@@ -43,13 +43,13 @@ StressDivergenceTensors::validParams()
       "The eigenstrain_name used in the ComputeThermalExpansionEigenstrain.");
   params.addCoupledVar("out_of_plane_strain",
                        "The name of the out_of_plane_strain variable used in the "
-                       "WeakPlaneStress kernel. Required only if want to provide off-diagonal "
-                       "Jacobian in plane stress analysis using weak formulation.");
+                       "WeakPlaneStress kernel.");
   MooseEnum out_of_plane_direction("x y z", "z");
   params.addParam<MooseEnum>(
       "out_of_plane_direction",
       out_of_plane_direction,
       "The direction of the out_of_plane_strain variable used in the WeakPlaneStress kernel.");
+  params.addCoupledVar("thickness", "Out of plane thickness for plane stress/plane strain models");
```

Comment:
Is this coupled variable used?

Reply:
@hugary1995 Thanks for catching that! I had added an option to have the user provide an out-of-plane thickness as a variable. This would allow it to either be specified through an AuxVariable for non spatially uniform cases, or as a constant value. However, I realized that it's a little more complicated than just adding it to this one Kernel because we might want to integrate over volumes in other places in the code, so it should be added at a lower level. Also, in a lot of places we actually want an area, rather than a volume integral, so I'm not really sure what to do about that. I'm sticking with our current assumption of a unit thickness for the time being.

Reply:
I pushed an update that deletes this line, since you're right -- it isn't used in my current version of the code.